### ISSUE TITLE
DailyRewardGui sometimes throws CMEs on initGui

### ISSUE THREAD

**Issue by robere2** [bugfroggy/Quickplay2.0#49]
`initGui` calls `componentList.clear()`. It seems like `DailyRewardGui` might be accessing the `componentList` somewhere while that's happening which is causing a crash. Fortunately it doesn't happen often. Unclear if this is occurring in other GUIs as well.

Attempted to fix this in commit dd7c276 but ultimately it didn't work I don't believe.

**Comment by robere2** [comment]
Stacktrace:
```
java.util.ConcurrentModificationException
	at java.util.ArrayList$Itr.checkForComodification(ArrayList.java:901)
	at java.util.ArrayList$Itr.next(ArrayList.java:851)
	at co.bugg.quickplay.client.gui.QuickplayGui.func_73863_a(QuickplayGui.java:226)
	at co.bugg.quickplay.premium.dailyreward.DailyRewardGui.func_73863_a(DailyRewardGui.java:354)
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:311)
	at sun.reflect.GeneratedMethodAccessor34.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at Reflector.callVoid(Reflector.java:509)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1430)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1051)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:349)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```
Caused by `componentList.clear()`

**Comment by robere2** [comment]
This seems to be related to #57

**Comment by robere2** [comment]
This might be fixed in commit bugfroggy/QuickplayPremium@fd89844.

**Comment by robere2** [comment]
Closing until further notice. This should be fixed.
